### PR TITLE
Feature/past jobs

### LINKED ISSUE #193: [Profile] Jobs Section | Add previous jobs
Hey! This is a pretty awesome app :smile:

I feel like adding a 'previous jobs' section would be nice. For example, on the mentorship page, selecting "professional developers only" doesn't show anyone who isn't currently employed. But you still might have worked professionally (hence, you're a professional).

Hope that makes sense.

### PR DESCRIPTION
**Pull request by AkiraLaine**
closes #193 
This is pretty much what I showed on the issue. If you still want me to change something, let me know.

### CODE REVIEW

**Comment by no-stack-dub-sack** [comment]
Hey @AkiraLaine, thanks for this! I've had a busy day today, let me give this a more careful review tomorrow, but at first glance it looks good to me. Have you tested out the UI for adding several companies, though? I wonder how that will look on the Public Profile page. 

Also, we need to consider how this affects the page validations for this section. 

When I look at this tomorrow I'll pull down the code and run things so I can see how it looks a bit closer up and test things out. However, I'm leaving a quick first glance review because I see at least 1 change that will need to be made.

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,2694 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (no-stack-dub-sack) [inline yarn.lock]: ☝️ looks like this will have to be added to the `.gitignore`

**Comment by AkiraLaine** [comment]
You can add several companies in the input field (I added a 'separated by a comma' text for readability when it gets displayed).

As for the `yarn.lock` file, some people might be using `yarn` instead of `npm` (like me), doesn't make sense to add it to the `.gitignore`.

Once you're happy with everything and we made any last minute changes, I'll squash everything :smile:

**Comment by no-stack-dub-sack** [comment]
@AkiraLaine Ohhhh, ok. I see what it is now. NPM actually has a similar file in their new NPM 5 release called `package.lock`. That's cool then. We can leave it. Going to pull down the changes now. 

I saw your comment with the image, but what I was asking is what if someone adds like 10 companies - how will that render on the public profile page? I will test it shortly.

**Comment by AkiraLaine** [comment]
Okay, I'll make sure a user needs to fill out all inputs.

As for a user entering 10 companies, what I'm thinking is that we change the phrase to "Enter companies separated by a comma (max: 3)". And then just have a validation that turns the string into an array and checks if length <= 3

**Comment by AkiraLaine** [comment]
I ended up working on the changes. Please tell me what you think.

**Comment by no-stack-dub-sack** [comment]
@AkiraLaine This looks perfect. I agree with your choices. Thanks a ton for the work and hope you feel better - I'm still getting over a sickness myself. I can def relate. 

If you want to squash your commits since we're all set on changes I'll merge this in!

**Comment by AkiraLaine** [comment]
@no-stack-dub-sack I believe there's a dropdown next to 'merge pull request' that allows you to squash and merge.

I can't do it locally since I've pulled from master during development. :/

**Comment by no-stack-dub-sack** [comment]
@AkiraLaine Yeah, there is, that's fine then. I can do it on this end. There might be a slight difference in how it shows up in the commit history but I'm not particularly worried about it. Thanks again!

File: client/src/components/dashboard/Profile/Public/CareerRow.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const CareerRow = ({ career }) => {
         <TableRow
           header="I have been coding for:"
           content={career.tenure} />
-        {career.hasBeenEmployed &&
+        {career.hasBeenEmployed === 'yes' &&
```
Comment (no-stack-dub-sack) [inline client/src/components/dashboard/Profile/Public/CareerRow.js]: @AkiraLaine nice last minute catch!

**Comment by no-stack-dub-sack** [comment]
@AkiraLaine Ok, I'm redeploying now - should be just a few minutes. Maybe just give this some quick testing on the prod site to make sure everything is working as expected when you get around to it (I don't see why it wouldn't). I won't be able to check things out for a little while - about to hit the road for a few hours.

**Pull request merged**